migrate_game.rb

```diff
@@ -47,6 +47,11 @@ def repair(game, original_actions, actions, broken_action)
     # Move token is now place token.
     broken_action['type'] = 'place_token'
     return [broken_action]
+  elsif game.is_a?(Engine::Game::G1889::Game) || game.is_a?(Engine::Game::G18GB::Game)
+    if game.active_step.is_a?(Engine::Step::BuyCompany) || game.active_step.is_a?(Engine::Step::TrackAndToken)
+      add_pass.call
+    end
+    return
   elsif game.is_a?(Engine::Game::G18USA::Game)
```

Comment:
because of the fix to corp.all_abilities, 1889 & 18gb will have extra passed needed, when a player is prompt to use the ability